ci-operator/step-registry/baremetalds/ofcir/pre/baremetalds-ofcir-pre-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: baremetalds-ofcir-pre
+  steps:
+  - ref: baremetalds-ofcir-setup
+  - ref: ipi-install-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: baremetalds-devscripts-proxy
+  - ref: baremetalds-devscripts-setup
```

Comment:
This coupling seems to me too strong. Not necessarily an ofcir user may want to use dev-scripts to setup his/her own environment. I think we should really keep the ofcir steps to their bare minimum